server/channels/app/slashcommands/command_invite_people.go

```diff
@@ -16,15 +16,15 @@ type InvitePeopleProvider struct {
 }
 
 const (
-	CmdInvite_PEOPLE = "invite_people"
+	CmdInvitePEOPLE = "invite_people"
```

Comment:
Why not
```suggestion
	CmdInvitePeople = "invite_people"
```
?

Reply:
This is an easy fix, I'll implement this.

Reply:
implemented.